cli/cmd/compose/pull.go

```diff
@@ -57,6 +59,15 @@ func runPull(ctx context.Context, opts pullOptions, services []string) error {
 		return err
 	}
 
+	if !opts.includeDeps {
+		enabled, err := project.GetServices(services...)
+		if err != nil {
+			return err
+		}
+		project.DisabledServices = append(project.DisabledServices, project.Services...)
```

Comment:
DisabledServices will contain all services including the enabled ones here. Not affecting the current pull execution path probably, but very confusing, we should remove enabled services from there

Reply:
ok, I'll fix the other places we do the same for "noDeps"

Reply:
see https://github.com/docker/compose-cli/pull/1377/commits/42d1dd2e0d0d7ce3d77e384445b79eb93905fda7